modules/apps/fragment/fragment-impl/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why do we need a `package.json` here?

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/268/commits/93a06511f228c183a9363f8a65219ad588c3c00a

> Note, also, that we attach a new package.json. This is needed so that NPMRegistry
is aware of the `fragment-impl` JSBundle so that we can attach fragments'
JSModules to it.

Reply:
![knowitall](https://user-images.githubusercontent.com/905006/90871293-23e70480-e39b-11ea-83bc-f51ce3149250.gif)

Reply:
> Note, also, that we attach a new package.json. This is needed so that NPMRegistry
is aware of the fragment-impl JSBundle so that we can attach fragments'
JSModules to it.

It is weird that the `fragment-impl` module as a `package.json`... could this live elsewhere? same as with the ModelListener?

Reply:
Yes, it could live in a different module, but because of [this](https://github.com/liferay-frontend/liferay-portal/pull/268#discussion_r474543521) we should make sure that, if the react support module is not deployed, the fragment renderer won't fail (that can be done with OSGi APIs, I think...).